### ISSUE TITLE
Alt Text Saved to Image in Media Library Not Used When Added to Post or Page

### ISSUE THREAD

**Issue by mdrockwell** [wordpress-mobile/gutenberg-mobile#4949]
**Describe the bug**
Alt text can be saved on an image within the media library and when adding that image to a post or page within the browser, the alt text is properly applied. This isn't the behavior in the mobile app.

**To Reproduce**
1. Navigate to My Site > Menu > Media.
2. Select an image from your media library and add alt text.
3. Save the changes.
4. Create a new post or page within the app and insert the image.

**Expected behavior**
The alt text saved within the media library should be added to the image, but it is not.

**Smartphone (please complete the following information):**
 - Device: Pixel 3
 - OS: Android 12
 - Version 20.0-rc-2

 - Device: iPhone 13 Pro
 - OS: iOS 15.5
 - Version 20.0.0.1

**Additional context**
5285114-zd-woothemes

**Comment by reginabally** [comment]
Another report in 5469105-zd-woothemes.

Device: iPhone 12 (iPhone13,2)
OS: 15.5
Version: 20.2.1

**Comment by dcalhoun** [comment]
@reginabally @mdrockwell I was able to reproduce this using the WP 20.2.1 app. However, I was **not able** to reproduce this on the latest public release: 20.4.0.3. I tested an iPhone SE (iOS 15.6) and Samsung Galaxy S20 (Android 12). 

May we recommend users upgrade to the latest public release to see if this resolves the issue for them? 

Closing this, but feel free to reopen if needed.
